src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -305,7 +304,37 @@ public Number[] executeSimple(Object[][] params) {
 		}.description("Returns the minimum number from a list of numbers.")
 			.examples("min(1) = 1", "min(1, 2, 3, 4) = 1", "min({some list variable::*})")
 			.since("2.2"));
-		
+
+		Functions.registerFunction(new SimpleJavaFunction<Number>("clamp", new Parameter[]{
+			new Parameter<>("values", DefaultClasses.NUMBER, false, null),
+			new Parameter<>("min", DefaultClasses.NUMBER, true, null),
+			new Parameter<>("max", DefaultClasses.NUMBER, true, null)
+		}, DefaultClasses.NUMBER, false) {
+			@Override
+			public @Nullable Number[] executeSimple(Object[][] params) {
+				Number[] values = (Number[]) params[0];
+				Double[] clampedValues = new Double[values.length];
```

Comment:
Could this be the primitive type? `double[]`

Reply:
I don't think so, it wants a Number[] as the return value so Double[] is fine but double[] is not.

Reply:
Ohh, you're right. it's fine how it is now then